### DISCUSSION TITLE
SW_STORAGE_ES_INDEX_REPLICAS_NUMBER config failed

### DISCUSSION THREAD

**Discussion by ms787277296** [apache/skywalking#11774] [Q&A] [answered]
the version of oap i used is 9.7, when i set SW_STORAGE_ES_INDEX_REPLICAS_NUMBER = 1 and start oap service, only some es indies created their replications. it seems that this config has no effect on the sw_segment and sw_log indies
![img_v3_0274_9e566b01-8cbb-4972-8054-b41a43a91f3g](https://github.com/apache/skywalking/assets/29354555/9305ddf4-492e-4995-846d-ec8021d893cc)

**Comment by wu-sheng** [comment, marked as answer]
Check this doc, https://skywalking.apache.org/docs/main/next/en/setup/backend/storages/elasticsearch/#index-settings

These two are different.
